src/components/button.component.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Button = (props) => (
+        <button className={`${props.wide}-button-${props.color}`}>{props.button}</button>
+)
+
+Button.propTypes = {
```

Comment:
- [ ] buttons should take 25% container width except the `0` button. Add styling to the button component. You can declare a `button.css` file with the styles needed.
- [ ] Buttons should have borders
- [ ] Button's text should be centered and black in color
- [ ] Buttons should have default color of orange
- [ ] `Wide` is boolean, true means that button take 50% of space (used for 0 button)